spark-3.5.yaml

```diff
@@ -63,18 +62,105 @@ pipeline:
 
   - uses: maven/pombump
 
-  - name: build
-    runs: |
-      ./dev/make-distribution.sh --name custom-spark --pip -Psparkr -Phive -Phive-thriftserver -Pmesos -Pyarn -Pkubernetes
+data:
+  - name: openjdk-versions
+    items:
+      8: "java-1.8-openjdk"
+      11: "java-11-openjdk"
+      17: "java-17-openjdk"
```

Comment:
Does Spark run with OpenJDK 21? If so, we should add it here

Reply:
I don't think they "official" have support on `21`:

https://spark.apache.org/docs/latest/
> Spark runs on Java 8/11/17

And also no Java 21 on their [versions.json](https://github.com/apache/spark-docker/blob/master/versions.json).